internal/store/postgres/servicedata.go

```diff
@@ -20,16 +21,26 @@ type Key struct {
 }
 
 type ServiceData struct {
-	URN         string `db:"urn"`
-	NamespaceID string `db:"namespace_id"`
-	EntityID    string `db:"entity_id"`
-	Value       string `db:"value"`
-	Key         string `db:"key"`
-	ProjectID   string `db:"project_id"`
-	ResourceID  string `db:"resource_id"`
+	URN         string         `db:"urn"`
+	NamespaceID string         `db:"namespace_id"`
+	EntityID    string         `db:"entity_id"`
+	Value       sql.NullString `db:"value"`
+	Key         string         `db:"key"`
```

Comment:
Probably we could be more specific on this one, whether this is `KeyURN` or `KeyID` or `KeyName`